tests/functional/generate_pages.py

```diff
@@ -153,7 +160,7 @@ def process_answer(question_type, answer, page_spec):
         logger.debug("\t\tSkipping Child Answer: %s", answer['id'])
         return
     elif answer['type'] == 'Radio' or answer['type'] == 'Checkbox':
-        process_options(answer['id'], answer['options'], CHECKBOX_RADIO_CLICKER, page_spec)
+        process_options(answer['id'], answer['options'], CHECKBOX_RADIO_CLICKER, CHECKBOX_RADIO_IS_SELECTED, page_spec)
```

Comment:
Looks like answer_id is used on line 139 directly as well but I'll make some improvements to this as there is some duplication